docs/osf-trivia-list/dasharo.md

```diff
@@ -0,0 +1,220 @@
+hardware market understand only sales volumes nothing else.
+
+We want to work on changing above paradigms or at least improve state of the
+art relation in community to the level above on which threats will have
+reasonably small impact. We believe that open-source firmware is critical tool,
+which should be used consciously to ensure privacy and liberty.
+
+## What is open-source firmware distribution?
+
+Dasharo is 3mdeb firmware distribution and all its components are open-source.
+We provide releases in binary form. As you know in most cases on x86 for
+firmware do be useful it has to cooperate with closed blobs. In all binary
+releases we made we provide information where all components coming from.
+
+Dasharo works without blobs on platforms that allow that. When we saying
+Dasharo open-source firmware distribution we mean code that is delivered by
+3mdeb that is open-source e, we have no influence on code of 3rd parties (e.g.
+FSP, ME, GbE etc.).
+
+In coreboot community there was some controversy about calling Dasharo
+open-source firmware distribution (for details please check [gerrit review][gerrit-review].
+We respect coreboot community opinion, so we agreed that in case of coreboot
+documentation it would be better to use open-source based firmware
+distribution. It doesn't mean we agree with that decision:
+* Definitions and rules used in coreboot documentation review
+* Rules seem to no be applied equally to all contributors of [coreboot
+distribution][cb-distro]
+
+## Why Dasharo is not called coreboot firmware distribution?
+
+Dasharo is not coreboot firmware distribution because we also may imagine
+Dasharo pure EDKII, oreboot or based on any other open-source firmware
+framework. Dasharo typically is based on coreboot and EDKII, plain coreboot is
+not enough in most cases for booting modern computer.
+
+## What value Dasharo provide in comparison to coreboot?
+
+* Dasharo is open-source firmware distribution based on coreboot and other
+open-source firmware frameworks, you can think about Dasharo and coreboot
+relation in the same way as you think about Debian/Ubuntu/RedHat and Linux.
+* We provide long term maintenance - coreboot community for various reasons, do
+not merge some patches, because understaffing, lack of reviewer some changes
+have long way to upstream, we maintain those patches and make them work
+before those will go upstream, if ever, we are committed to maintain platforms
+which are moved to branch in coreboot.
+* We provide transparent validation results - coreboot in itself provide no
+guarantees around release quality and do not provide binary distribution (for
+reference please check [coreboot project scope][coreboot-scope], we provide
+those in scope of validation we perform.
+* We provide seamless deployment - our documentation provide information about
+initial deployment, updates and recovery procedures. We working on commercial
```

Comment:
```suggestion
initial deployment, updates and recovery procedures. We working on a
```

Is `commercial` really needed here?

Reply:
No longer needed. It is already slightly outdated.